webapp/src/views/CrudList.vue

```diff
@@ -576,9 +575,6 @@ export default {
               typeof value === 'string' &&
               value.toString().toLocaleLowerCase().indexOf(search.toLocaleLowerCase()) !== -1
     },
-    getDatatype(datatype, item) {
```

Comment:
this is important for editing of RRs

Reply:
but it's currently dead, and I ran into some issue when refactoring the datatable `item.<name>` slots. I decided to fix it when introducing RRset editing.